assets/js/base/components/cart-checkout/order-summary/order-summary-item.js

```diff
@@ -60,7 +62,12 @@ const OrderSummaryItem = ( { cartItem } ) => {
 						value={ linePrice }
 					/>
 				</div>
-				<ProductLowStockBadge lowStockRemaining={ lowStockRemaining } />
+				{ ! showBackorderBadge && (
+					<ProductLowStockBadge
+						lowStockRemaining={ lowStockRemaining }
+					/>
+				) }
+				{ showBackorderBadge && <ProductBackorderBadge /> }
```

Comment:
I see from this logic that the two badges have different behaviour/interface:

- low stock badge auto hides itself when appropriate
- backorder badge always renders something, so the client component needs to control render directly

I can see why this is the case - `lowStockRemaining == 0`, but it might be clearer to use ternary logic here, e.g.

```
showBackorderBadge ? <ProductBackorderBadge /> : lowStockRemaining && <ProductLowStockBadge lowStockRemaining={ lowStockRemaining } />
```

🤔 

Or maybe that's more confusing! Not a blocker just sharing an observation - totally happy with the code as is.

Reply:
I like it! Prettier breaks it into several lines, so I don't think it's more confusing at all. Changed in: e635ecfa207030cdbd103fd2ec0e58249c8a2a7f.